engine/apps/user_management/models/organization.py

```diff
@@ -181,9 +163,17 @@ def _get_subscription_strategy(self):
         "slack.SlackTeamIdentity", on_delete=models.PROTECT, null=True, default=None, related_name="organizations"
     )
 
-    # Slack specific field with general log channel id
+    # TODO: drop this field in a subsequent release, this has been migrated to general_log_slack_channel field
     general_log_channel_id = models.CharField(max_length=100, null=True, default=None)
 
+    general_log_slack_channel = models.ForeignKey(
```

Comment:
nit: suggest renaming this field to `default_slack_channel` or `general_slack_channel`. wdyt?

Reply:
good suggestion 👍 updated in https://github.com/grafana/oncall/pull/5191/commits/ce5a0978ccc151dd9b87a32b027a4ce8bf84fb79